docs/docs/en/guide/installation/pseudo-cluster.md

```diff
@@ -83,6 +83,8 @@ each machine. You could find this file in the path `bin/env/install_env.sh` and
 # ---------------------------------------------------------
 # Due to the master, worker, and API server being deployed on a single node, the IP of the server is the machine IP or localhost
 ips="localhost"
+# Modify Configurationexport sshPort="22"
```

Comment:
Not this place, I mean we should maybe add some sentences in L79 or L144, to tell users they can change env variable by `export <ENV_NAME>=<VALUE>`

Reply:
Others LGTM